open-sgid-database/index.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Connect to the Open SGID
+author:
+  display_name: AGRC Office
+  email: agrc@utah.gov
+date: 2020-05-26 13:58:58 -0600
+layout: page_nocrumb
+categories: []
+tags:
+  - how to
+  - sgid
+  - open sgid
+---
+
+The Open SGID database is a PostGIS Cloud SQL mirror of the AGRC SGID database. This database compliments the web services and data downloads offered on our [Open Data site](https://opendata.gis.utah.gov/). 
+
+To search the SGID, please visit the [SGID Index]({% link data/sgid-index/index.html %}). To learn more about the SGID and the data, please visit the [AGRC data page]({% link data/index.html %}).
+
+### Before you Connect...
+{: .text-left }
+
+- Please take a moment to read over the [Open SGID Terms of Use]( {% link about/policy/open-sgid/index.md %}).
+- Get familiar with the [Open SGID GitHub repository](https://github.com/agrc/open-sgid) where you can troubleshoot and track known issues.
+
+### How to Connect
+{: .text-left }
+
+There are a variety of clients and application that connect to a PostgreSQL/PostGIS database. Here's how to connect using a few of the more common applications (ArcGIS Desktop, ArcGIS Pro, DBeaver, and QGIS). 
```

Comment:
Add _"s"_ to pluralize _applications_ in the first sentence

Reply:
tip: use the suggestion feature for little modifications like this.